public/app.css

```diff
@@ -124,11 +123,15 @@ button:focus {
 #emoji-text {
   text-align: center;
   text-transform: uppercase;
-  color: #fff;
+  color: transparent;
   font-size: 10em;
-  font-family: "Helvetica Neue", Ubuntu, sans-serif;
+  font-family: "Avenir Next Condensed", "Helvetica Neue", Ubuntu, sans-serif;
   font-weight: bold;
   line-height: 100%;
+  text-shadow: 
+  0.01em 0.01em 0.02em #000,
+  0 0 0 rgba(255, 255, 255, 0.75);
+  filter: invert(1);
```

Comment:
Wow, that is fancy.

Reply:
for some reason I couldn't figure out how to get it working without using the invert